lib/nx_gzlib.c

```diff
@@ -103,3 +103,24 @@ gzFile nx_gzopen_(const char* path, int fd, const char *mode)
 	file = (gzFile) state;
 	return file;
 }
+
+int nx_gzwrite (gzFile file, const void *buf, unsigned len)
+{
+	gz_statep state = (gz_statep) file;
+	z_streamp stream = &(state->strm);
+	int rc;
+	uLong last_total_in = stream->total_in;
+	unsigned char* next_out = malloc(sizeof(char)*len);
```

Comment:
Where is this being freed?

Reply:
It isn't, I'll fix that for the next version.